network-api/networkapi/buyersguide/views.py

```diff
@@ -28,7 +31,19 @@ def get_average_creepiness(product):
     return 50
 
 
+def path_is_en_prefixed(path):
+    return path.startswith('/en/')
+
+
+def get_en_redirect(path):
+    redirect_path = re.sub(path_regex, '/en/', path, count=1)
+    return redirect(redirect_path, permanent=False)
+
+
 def buyersguide_home(request):
+    if not path_is_en_prefixed(request.path):
```

Comment:
This feels like it wants to be a decorator, given how many places we repeat this code in.